R/zzz.R

```diff
@@ -0,0 +1,6 @@
+.onAttach <- function(libname, pkgname) {
+    msg <- sprintf(
+        "Package '%s' is deprecated and will be removed from Bioconductor
```

Comment:
Have you @sneumann  checked with Christoph on that?

He reached out recently to ask if I could update IPO, but I do not have time for that. If the idea is to continue to offer IPO, we'd need to find a new place for it.